ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -80,19 +80,20 @@ export const newClusterDefault = {
     collectionMethod: defaultCollectionMethod,
     DEPRECATEDProviderMetadata: null,
     admissionControllerEvents: true,
-    admissionController: false,
-    admissionControllerUpdates: false,
+    admissionController: true, // default changed in 4.9
+    admissionControllerUpdates: true, // default changed in 4.9
+    admissionControlFailOnError: false, // property added in 4.9 false means Fail open
```

Comment:
What does the "open" mean? Typo?

Reply:
Oh, I saw something similar in a different file. What do "Fail open" and "Fail closed" mean?